lib/jxl/color_encoding_internal.cc

```diff
@@ -718,8 +718,12 @@ static const float kBradfordInv[9] = {
 
 // Adapts whitepoint x, y to D50
 Status AdaptToXYZD50(float wx, float wy, float matrix[9]) {
+  if (wx < 0 || wx > 1 || wy <= 0 || wy > 1) {
```

Comment:
what about float numbers that are veeery close to 0?

Reply:
I tested 1 / 5e-324 (smallest float) and indeed that gives infinity, added back the std::finite checks